src/parser/utilities.hh

```diff
@@ -20,59 +20,54 @@
 
 namespace rtt_parser {
 //! Can the next token in the stream be interpreted as real number?
-DLL_PUBLIC_parser bool at_real(Token_Stream &tokens);
+bool at_real(Token_Stream &tokens);
 
 //! Is the next token in the stream a unit name?
-DLL_PUBLIC_parser bool at_unit_term(Token_Stream &tokens,
-                                    unsigned position = 0);
+bool at_unit_term(Token_Stream &tokens, unsigned position = 0);
 
-DLL_PUBLIC_parser unsigned parse_positive_integer(Token_Stream &);
+unsigned parse_positive_integer(Token_Stream &);
 
-DLL_PUBLIC_parser unsigned parse_unsigned_integer(Token_Stream &);
+unsigned parse_unsigned_integer(Token_Stream &);
 
-DLL_PUBLIC_parser int parse_integer(Token_Stream &);
+int parse_integer(Token_Stream &);
 
-DLL_PUBLIC_parser double parse_real(Token_Stream &);
+double parse_real(Token_Stream &);
 
-DLL_PUBLIC_parser double parse_positive_real(Token_Stream &);
+double parse_positive_real(Token_Stream &);
 
-DLL_PUBLIC_parser double parse_nonnegative_real(Token_Stream &);
+double parse_nonnegative_real(Token_Stream &);
 
-DLL_PUBLIC_parser bool parse_bool(Token_Stream &);
+bool parse_bool(Token_Stream &);
 
-DLL_PUBLIC_parser Unit parse_unit(Token_Stream &);
+Unit parse_unit(Token_Stream &);
 
-DLL_PUBLIC_parser void parse_vector(Token_Stream &, double[]);
+void parse_vector(Token_Stream &, double[]);
 
 //! parser a quote-delimited string, stripping the quotes.
-DLL_PUBLIC_parser std::string parse_manifest_string(Token_Stream &tokens);
+std::string parse_manifest_string(Token_Stream &tokens);
 
-DLL_PUBLIC_parser void
-parse_geometry(Token_Stream &tokens,
-               rtt_mesh_element::Geometry &parsed_geometry);
+void parse_geometry(Token_Stream &tokens,
+                    rtt_mesh_element::Geometry &parsed_geometry);
 
-DLL_PUBLIC_parser void parse_unsigned_vector(Token_Stream &, unsigned[],
-                                             unsigned);
-DLL_PUBLIC_parser void
-set_internal_unit_system(rtt_units::UnitSystem const &units);
-DLL_PUBLIC_parser void set_unit_expressions_are_required(bool);
-DLL_PUBLIC_parser rtt_units::UnitSystem const &get_internal_unit_system();
-DLL_PUBLIC_parser bool unit_expressions_are_required();
-DLL_PUBLIC_parser void free_internal_unit_system();
+void parse_unsigned_vector(Token_Stream &, unsigned[], unsigned);
+void set_internal_unit_system(rtt_units::UnitSystem const &units);
+void set_unit_expressions_are_required(bool);
+rtt_units::UnitSystem const &get_internal_unit_system();
```

Comment:
AppVeyor is balking at `get_internal_unit_system()` for some reason.  Does the `DLL_PUBLIC_parser` prefix need to be removed from the definition in `parser/utilities.cc` as well?

Reply:
Let me fix this before it is merged.

Reply:
I've pushed a patch. Let's see if Appveyor likes it now.